.gitignore

```diff
@@ -20,3 +20,4 @@ log.*
 *.csr
 *.srl
 *_creds
+cp-quickstart/docker-compose.yml
```

Comment:
FYC: It might be easier across our many sub-projects to keep `.gitignore` files in subfolders themselves instead of centrally at the root.   Also might be better / easier if we end up moving / or deprecating projects if they have their own .gitignore.

Reply:
Great idea @rspurgeon .  Done